src/ui.cpp

```diff
@@ -538,20 +543,21 @@ void uilist::calc_data()
             if( entries[ i ].retval == -1 ) {
                 entries[ i ].retval = i;
             }
-            if( w_auto && w_width < txtwidth + pad + 4 + clen ) {
-                w_width = txtwidth + pad + 4 + clen;
+            if( w_auto && w_width < txtwidth + pad + WIDTH_BUFFER + clen ) {
+                w_width = txtwidth + pad + WIDTH_BUFFER + clen;
             }
         } else {
-            if( w_auto && w_width < txtwidth + pad + 4 + clen ) {
+            if( w_auto && w_width < txtwidth + pad + WIDTH_BUFFER + clen ) {
                 // TODO: or +5 if header
-                w_width = txtwidth + pad + 4 + clen;
+                w_width = txtwidth + pad + WIDTH_BUFFER + clen;
             }
         }
         if( desc_enabled ) {
-            const int min_desc_width = std::min( max_desc_width, std::max( w_width, descwidth_final ) - 4 );
+            const int min_desc_width = std::min( max_desc_width, std::max( w_width,
+                                                 descwidth_final ) - WIDTH_BUFFER );
             int descwidth = find_minimum_fold_width( footer_text.empty() ? entries[i].desc : footer_text,
                             desc_lines, min_desc_width, max_desc_width );
-            descwidth += 4; // 2x border + 2x ' ' pad
+            descwidth += WIDTH_BUFFER; // 2x border + 4x ' ' pad
```

Comment:
Might not be needed because description does not start with a hotkey hint.

Reply:
resolved by https://github.com/CleverRaven/Cataclysm-DDA/pull/64478/commits/2fbc662ed6e32eee9b009acc12f23fd661beb2c0